configure.ac

```diff
@@ -0,0 +1,116 @@
+# Configure template for Gerbil system.
```

Comment:
NB: I don't speak autoconf and will assume you did the right thing here.

Reply:
It's my first time speaking autoconf. But, I ripped off gambit, so it's at least somewhat working.